### DISCUSSION TITLE
luci-adguardhome not found

### DISCUSSION THREAD

**Discussion by lostmaniac** [immortalwrt/immortalwrt#954] [General]
### Describe the bug

Dear mirror repository provider,

Hello, I am one of your users. I appreciate your mirror repository very much, which allows me to install and update various packages easily.

Recently, I wanted to install adguardhome on my router, a software for ad blocking and privacy protection. I found the basic package of adguardhome in your mirror repository, but I did not find the package of luci-adguardhome, which is a graphical interface for configuring and managing adguardhome.

I would like to ask you, is this because I used it incorrectly, or because your mirror repository does not provide the package of luci-adguardhome? If it is the latter, do you have any plans to add this package to your mirror repository?

I look forward to your reply. Thank you for your time and help.

### ImmortalWrt version

r20007-6e54cb5d37

### ImmortalWrt target/subtarget

rockchip/armv8

### Device

FriendlyElec NanoPi R2S

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
No, and never. That buggy luci app written in lua is a whole crap and has many security issues. It also lacks proper maintenance.
